crates/wasi-crypto/src/lib.rs

```diff
@@ -0,0 +1,33 @@
+pub use wasi_crypto::WasiCryptoCtx;
+
+mod witx {
+    pub use wasi_crypto::wasi_modules::*;
+}
+
+wasmtime_wiggle::wasmtime_integration!({
+    target: witx,
+    witx: ["$CARGO_MANIFEST_DIR/../../crates/wasi-crypto/spec/witx/wasi_ephemeral_crypto.witx"],
```

Comment:
This path seems a bit odd, but since it's crate relative I think everything before `spec` can be dropped?

Reply:
Absolutely. Done! Thank you!